classroom.rb

```diff
@@ -0,0 +1,13 @@
+class Classroom
+  attr_accessor :label, :students
```

Comment:
- [ ] For @students it is not needed to have attribute write access, for security reasons please make sure it only is added for read access.